### PR TITLE
feat(resources): Taku's auto CLI and GmsCore

### LINKED ISSUE #11: Suggestions / Issues for the (YT) step-by-step guide
1. On the [Getting the files ready](https://sodawithoutsparkles.github.io/revanced-troubleshooting-guide/step-by-step/03-get-files/) page, there is no info regarding the special GmsCore download for Hauwei devices. The reference image should also be updated to reflect this.

2. Do you want to add links Taku's auto cli for users who are on armeabi-v7a or don't have android 8?

3. On the [patching page](https://sodawithoutsparkles.github.io/revanced-troubleshooting-guide/step-by-step/07-patching/) and on the post-patching page (step 8), some images don't render properly on desktop firefox (i didn't test chrome)

Example:
image

### PR DESCRIPTION
**Pull request by SodaWithoutSparkles**
fixed #11

### CODE REVIEW

**Comment by SodaWithoutSparkles** [comment]
Going to wait til the other branch finished, dont want to deal w/ merge conflicts

File: index.md [hunk 1]
```diff
@@ -15,5 +15,6 @@ This is a guide on solving common issues when using ReVanced YT and ReVanced man
 ## Other resources
 
 - [Get notified for ReVanced Updates](https://discord.com/invite/cHHX7PSPux)
-<!-- - [Guide by my friend Hokora Yinphine MPP#6969 on discord](https://hokorayinphinempp.github.io/obsidian-git-sync/Revanced/1%20Start%20Here%21/Revanced%20Start/) -->
 - [Dummies guide on reddit](https://www.reddit.com/r/revancedapp/comments/xlcny9/revanced_manager_guide_for_dummies/)
+- [Taku's auto CLI to patch on computer](https://github.com/taku-nm/auto-cli)
+<!-- - [Guide by my friend Hokora Yinphine MPP#6969 on discord](https://hokorayinphinempp.github.io/obsidian-git-sync/Revanced/1%20Start%20Here%21/Revanced%20Start/) -->
```
Comment (KobeW50) [inline index.md:20]: Since it is outdated, it makes sense to add a note that it is outdated

Comment (SodaWithoutSparkles) [reply index.md:20]: you mean hokora's guide? It was commented out.

Comment (KobeW50) [reply index.md:20]: Oh lol. I didnt realize that's what that syntax meant

**Pull request merged**